client/src/components/ArticleList.jsx

```diff
@@ -4,11 +4,25 @@ import Article from './Article.jsx';
 class ArticleList extends Component {
   constructor(props) {
     super(props);
+    this.state = {
+      articles: this.props.articles
```

Comment:
i dont think setting props as state is a manageable pattern, iirc. i would just put the entire sortArticles function inside the render function before the return, and just continue to use the props

Reply:
i say this bc i think there will be strange collisions between the state of the page and the props being passed down. in the render function, all you would need is something like
`const articles = this.props.articles.sort(...).map(...)`